### PR TITLE
feat: use the new progress ux for the releasement of tasks

### PR DESCRIPTION
**Pull request by felixcicatt**
Consolidate the task releasement process to use the same UX as with cases.
Solves PZ-2542

### CODE REVIEW

File: src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts [hunk 1]
```diff
@@ -237,4 +255,61 @@ export class TakenWerkvoorraadComponent
     this.selection.clear();
     this.dataSource.filtersChanged();
   }
+
+  private handleAssigmentOrReleasementWorkflow<T>(
```
Comment (bas-info-nl) [inline src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts:259]: I would opt for `handleAssignOrReleaseWorkflow`, reason being the word Releasement is not used much anymore, and for both Assignment and Releasement we can use the base verbs, Assign and Release, to convey the same meaning.

Comment (bas-info-nl) [inline src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts:259]:
```suggestion
  private handleAssignOrReleaseWorkflow<T>(
```

Comment (bas-info-nl) [reply src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts:259]: And I notice a typo in `Assigment`, should be `Assignment`...

Comment (bas-info-nl) [reply src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts:259]: Or could this be created as 2 named types (aliases?) for the same underlying type? That would even be nicer IMHO 😸

**Review by bas-info-nl — APPROVED** [review]
I would prefer replacing use of `Releasement` with `Release`, and in similar vain replace `Assignment` with `Assign`. Please check my suggestions.

File: src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts (hunk 1 above)
Comment (edgarvonk) [reply src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts:259]: Agreed. In the backend we also use the word `release` to good to keep in sync.

ps: I never heard of the term `releasement` up till now. ;-)

**Review by edgarvonk — APPROVED** [review]

File: src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts (hunk 1 above)
Comment (felixcicatt) [reply src/main/app/src/app/taken/taken-werkvoorraad/taken-werkvoorraad.component.ts]: done :-)

**Pull request merged**